### DISCUSSION TITLE
UnidirectionalTopicOperator feature gate

### DISCUSSION THREAD

**Discussion by tai-nd** [strimzi/strimzi-kafka-operator#9043] [General]
Following the docs https://strimzi.io/docs/operators/latest/deploying#ref-operator-unidirectional-topic-operator-feature-gate-str
> For the KafkaTopic custom resource to use this feature, the strimzi.io/managed annotation is set to true by default.

Does that mean topics without the `strimzi.io/managed` annotation will be specified as bidirectional?

We have two Kafka clusters, one deployed on Kubernetes and managed by Strimzi, another deployed with cp-ansible, we want to use Strimzi TopicOperator to manage topics in both cluster, but we want to use bidirectional for the one created by strimzi and Unidirectional for another.

**Comment by tai-nd** [comment]
https://strimzi.io/docs/operators/latest/deploying#deploying-the-topic-operator-standalone-str

My bad :cry:

**Comment by scholzj** [comment]
Just to clarify -> you can use the Unidirectional Topic Operator with Strimzi powered Kafka cluster as well as with some other Kafka cluster. But they need to be separate installations with their own `KafkaTopic` resources. You should not share the same `KafkaTopic` resource with both of them.
